src/ssl.c

```diff
@@ -2160,8 +2160,6 @@ static int wolfSSL_read_internal(WOLFSSL* ssl, void* data, int sz, int peek)
     }
 #endif
 
-    sz = wolfSSL_GetMaxRecordSize(ssl, sz);
```

Comment:
Why remove the call to `wolfSSL_GetMaxRecordSize`? Should this be removed in all cases or maybe just for DTLS?

Reply:
`sz` is only used in `ReceiveData` to determine how much of the plaintext buffer to return to the user. MTU and record length checks are set and handled elsewhere. I don't see a reason why we should truncate the amount of data we return to the user. This is an old addition and might have been used to force an MTU limit but now that happens `SendData`.